recipe/conda_build_config.yaml

```diff
@@ -683,7 +681,7 @@ openexr:
 openh264:
   - '2.3'
 openjpeg:
-  - '2.4'
+  - '>=2.5,<3'
```

Comment:
For clarity, this is the new syntax I'm worried about breaking things

Reply:
I'm not sure this wouldn't break stuff. Let's keep this at `2.5` for now and change to 2 in a few months.

Reply:
Btw, I've changed this to 2 in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3410/files

Reply:
openjpeg is a "well behaved" packaged, likely, but it is possible that old builds are just "bad", with missing "pinnings" and do not have "run_exports".

I would prefer it if we could provide a lower bound, especially for things that are pinned to `'x`` only.